pkg/trustedresources/verify.go

```diff
@@ -40,11 +43,28 @@ const (
 
 // VerifyTask verifies the signature and public key against task.
 // source is from ConfigSource.URI, which will be used to match policy patterns. k8s is used to fetch secret from cluster
```

Comment:
I think this docstring could use an update to explain the function's new behavior with the feature flag